tests/integration.test.ts

```diff
@@ -0,0 +1,24 @@
+import * as BotD from '../src'
```

Comment:
A point of the "integration" test from FingerprintJS is to run the distributive script in the browsers. Here you run the source code. This is ok, but it doesn't make sure that the final code will work in the browsers because the source code doesn't include the changes made by Rollup.

Reply:
This notice is not resolved. You've made an IIFE bundle for integration testing, but haven't added it to this test. What happens [in FingerprintJS](https://github.com/fingerprintjs/fingerprintjs/blob/3a1d68080e573a8f05609716471598506c47834e/tests/integration.test.ts#L8) is that the agent types are taken from `src`, but the actual agent code is taken from a global namespace. The code is [build by Rollup](https://github.com/fingerprintjs/BotD/blob/e5e790c93c3ed0114446f6334e5afcaf4ee1dca7/rollup.config.test.js#L11) and [put to the global namespace by Karma](https://github.com/fingerprintjs/BotD/blob/e5e790c93c3ed0114446f6334e5afcaf4ee1dca7/karma.conf.js#L70).

Reply:
Thank you for the explanation! I fixed it.